### PR TITLE
Update 60-minute-blitz.jl

### PR DESCRIPTION
**Pull request by nicolasbarra**
Derivative no longer works, so gradient should be used here
Leads to confusion for beginners attempting to follow tutorial

### CODE REVIEW

**Comment by MikeInnes** [comment]
Thanks, but note that if you update the syntax you'll likely also need to update the project/manifest files.

**Comment by nicolasbarra** [comment]
> Thanks, but note that if you update the syntax you'll likely also need to update the project/manifest files.

What do you mean by this?

**Comment by MikeInnes** [comment]
See [these docs](https://julialang.github.io/Pkg.jl/dev/environments/). The `.toml` files in this folder provide a set of packages that work with this code. After your changes, the code works with a different (newer) set of packages.

**Comment by dehann** [comment]
Thanks for the fix, I just ran into this same issue on derivative.

**Comment by nicolasbarra** [comment]
> See [these docs](https://julialang.github.io/Pkg.jl/dev/environments/). The `.toml` files in this folder provide a set of packages that work with this code. After your changes, the code works with a different (newer) set of packages.

I doesn't seem like the packages will need to change, this is just a syntax change that happened in Flux and Flux is already a package there.

**Comment by MikeInnes** [comment]
The project records specific package _versions_. So if there's a syntax change in Flux, the project will load the older version with the old syntax, so it'll break if you change (only) the syntax.

**Comment by dhairyagandhi96** [comment]
You probably want to just activate the environment and call `] up` to get the newest versions of the packages in the environment.

**Comment by nicolasbarra** [comment]
I fixed the package/manifest files 
and made another request here:
https://github.com/FluxML/model-zoo/pull/148#issue-293389656

File: tutorials/60-minute-blitz.jl [hunk 1]
```diff
@@ -119,17 +119,17 @@ f(5)
...
 df(5)
 
 # You can try this with a few different inputs to make sure it's really the same
 # as `6x+2`. We can even do this multiple times (but the second derivative is a
 # fairly boring `6`).
 
-ddf(x) = derivative(df, x)
+ddf(x) = gradient(df, x)
 
 ddf(5)
```
Comment (mbeltagy) [inline tutorials/60-minute-blitz.jl]: This does not work on Julia 1.10 and Flux 0.8.3 
See also 
https://github.com/FluxML/Zygote.jl/issues/278

**Comment by hossein-pourbozorg** [comment]
The changes exist in #143, so should we close this PR?

**Comment by aminya** [comment]
@dhairyagandhi96 this can be closed as it fixed by #143

**Comment by dhairyagandhi96** [comment]
Closing since #143 is merged

**Pull request closed without merging**